lib/network_utils.ml

```diff
@@ -41,14 +41,19 @@ let igmp_query_maxresp_time = ref "5000"
 let enable_ipv6_mcast_snooping = ref false
 let mcast_snooping_disable_flood_unregistered = ref true
 
-let call_script ?(log_successful_output=false) ?(timeout=Some 60.0) script args =
+let call_script ?(log_successful_output=false) ?(timeout=Some 60.0) ?(detach_child=false) script args =
```

Comment:
I don't like the way that this function returns an empty string in the `detach_child` case.  It might be better to have two functions - `call_script`, which blocks and returns the output, and `fork_script` (or something similar) which detaches and returns unit.    These functions would share a lot of logic which could be factored out.   What do you think @robhoes ?

Reply:
I agree.